lib/protocol/src/openctx-protocol.schema.ts

```diff
@@ -44,6 +47,20 @@ export interface CapabilitiesResult {
      * At least 1 must be satisfied for the provider to be called. If empty, the provider is never called. If undefined, the provider is called on all resources.
      */
     selector?: Selector[]
+    /**
+     * Lists the kind of items the provider can return
+     */
+    itemKinds?: ItemKind[]
+    meta: {
```

Comment:
why is this required?

Reply:
If we need to list providers or even show that this context item is generated by this provider, we need the provider name. Provider URI isn't something we can show.

Reply:
I guess this will all depend on how this information is shown to the user. @sqs has been working on the user facing stuff the most here so I respect his opinion the most here :)